app/models/tokenModel.js

```diff
@@ -3,23 +3,13 @@ const Schema = mongoose.Schema;
 
 const PortalTokenSchema = new mongoose.Schema(
```

Comment:
From my understanding, the PortalToken only saves the token generated why the nick_name, email etc. saves in this data structure?

Reply:
We just want to record more content to enrich the token model. If these fields are not necessary, we can remove them.

Reply:
A reasonable explanation is required for this. Why do you need to record more content to enrich the token model, what's the tradeoff and consideration behind the scene?